### PR TITLE
Low-Code Apps section overhauled

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: user-guide/Advanced_Modules/Application_framework/Application_framework.md [hunk 1]
```diff
@@ -4,17 +4,12 @@ uid: Application_framework
...
-
-- [Configuring custom applications](xref:Configuring_custom_apps)
-
-- [Deleting a custom application](xref:Deleting_custom_apps)
+> [!IMPORTANT]
+> The Low-Code Apps module requires the "Low-Code Apps" license. For more information on acquiring this license, contact the Skyline Sales department.
 
 > [!NOTE]
-> The Application Low-Code Apps require the "Low-Code Apps" license. For more information on acquiring this license, contact the Skyline Sales department.
+> To view, add, edit, delete, or publish low-code applications, you need to have the necessary user permissions under [Modules > User-definable apps](xref:DataMiner_user_permissions#modules--user-definable-apps) as well as the [DataMiner web apps](xref:DataMiner_user_permissions#general--dataminer-web-apps--dataminer-cube-mobile-access) user permission.
```
Comment (SebastiaanSL) [inline user-guide/Advanced_Modules/Application_framework/Application_framework.md:15]: Are low-code applications and Low-Code Apps both fine to use? So one with capitals and the other without?

File: user-guide/Advanced_Modules/Application_framework/Low-Code_Apps_configuration/LowCodeApps_page_config.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+uid: LowCodeApps_page_config
+---
+
+# Configuring a page of a low-code app
+
+Custom low-code apps can consist of one or more pages that display the content of the application. These are similar to [DataMiner dashboards](xref:newR_D), though they include extra features and components that allow interaction with the displayed data. Data from a component on a page can be fed to components on the same page or on another page or panel, so that different pages and panels can influence each other.
```
Comment (SebastiaanSL) [inline user-guide/Advanced_Modules/Application_framework/Low-Code_Apps_configuration/LowCodeApps_page_config.md:7]: "Custom" low-code apps doesn't sound correct.

File: user-guide/Advanced_Modules/Application_framework/Low-Code_Apps_configuration/LowCodeApps_page_config.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+Each page can be configured with its own label and icon so users will easily be able to spot its purpose. You can also hide pages from the app sidebar, for instance because you only want them to be displayed if a user clicks a specific button.
```
Comment (SebastiaanSL) [inline user-guide/Advanced_Modules/Application_framework/Low-Code_Apps_configuration/LowCodeApps_page_config.md:9]: Their purpose?

**Review by SebastiaanSL — APPROVED** [review]
Looks good to me, my comments are negligible.

File: user-guide/Advanced_Modules/Application_framework/Application_framework.md (hunk 1 above)
Comment (MariekeGO) [reply user-guide/Advanced_Modules/Application_framework/Application_framework.md:15]: Indeed, we use "Low-Code Apps" when referring to the unique module, but low-code application when referring to the non-unique thing created using the module. This is based on the difference between a proper and common noun:
image
(See https://grammar.yourdictionary.com/capitalization/10-rules-of-capitalization.html)

**Pull request merged**